src/features/user/ProfileV2/components/MicroComponents/ProfileBox/UserInfo.tsx

```diff
@@ -0,0 +1,64 @@
+import myProfilestyle from '../../../styles/MyProfile.module.scss';
+import getImageUrl from '../../../../../../utils/getImageURL';
+import { useRouter } from 'next/router';
+import { Avatar } from '@mui/material';
+import { useTranslation } from 'next-i18next';
+import { ProfileProps } from '../../../../../common/types/profile';
+import { ReactElement } from 'react';
+import formatDate from '../../../../../../utils/countryCurrency/getFormattedDate';
+import Link from 'next/link';
+import {
+  DefaultProfileImage,
+  EditTargetSvg,
+} from '../../../../../../../public/assets/images/ProfilePageIcons';
+
+const UserInfo = ({ userProfile }: ProfileProps): ReactElement => {
+  const { t, ready } = useTranslation(['profile']);
+  const router = useRouter();
+
+  return ready ? (
+    <>
+      <div className={myProfilestyle.userInfoContainer}>
+        {userProfile?.image ? (
+          <Avatar
+            alt="user Image"
+            src={getImageUrl('profile', 'avatar', userProfile?.image)}
+            className={myProfilestyle.avatarRoot}
+          />
+        ) : (
+          <DefaultProfileImage />
+        )}
+
+        <div className={myProfilestyle.userInfoMainContainer}>
+          <div className={myProfilestyle.userInfo}>
+            {userProfile?.displayName}
+          </div>
+          <div>
+            {t('profile:myProfile.memberSince', {
+              date: `${formatDate(userProfile?.created)}`,
+            })}
+          </div>
+        </div>
+      </div>
+      {userProfile && router.asPath === '/profile' && (
+        <div className={myProfilestyle.iconContainer}>
+          <Link href="/profile/edit">
+            <div className={myProfilestyle.icon}>
+              <EditTargetSvg color="#007A49" />
+            </div>
+          </Link>
+        </div>
+      )}
```

Comment:
Shouldn't you check that the user has rights to edit this profile? i.e. that the profile of the logged in user matches this user?

Reply:
Resolved in 95d7ee004d329581628890c701b7d520167a048c